module/zfs/vdev_mirror.c

```diff
@@ -156,8 +161,8 @@ vdev_mirror_load(mirror_map_t *mm, vdev_t *vd, uint64_t zio_offset)
 	 * of the last I/O queued to this vdev as they should incur less
 	 * of a seek increment.
 	 */
-	if (ABS(lastoffset - zio_offset) <
-	    zfs_vdev_mirror_rotating_seek_offset)
+	offset_diff = lastoffset - zio_offset;
```

Comment:
Adding an explicit cast here would make it clear why it was necessary to add a new `offset_diff` variable.  Without it I could see how this change could be accidentally reverted if someone was trying to cleanup the code.

```
offset_diff = (int64_t)last_offset - (int64_t)zio_offset;
```